Changelog.md

```diff
@@ -4,6 +4,10 @@
 
 ## Common
 
+- Movie: Replace unprincipled "toeless sRGB" gamma function for PS1-derived movies with BT1886 Appendix 1 function with constants selected to match a properly calibrated mid-90s Sony Trinitron CRT.
+
+## Common
+
```

Comment:
Please compact this section and add the single new change line after the already existing one in the changelog ( top to bottom ). Thank you